.github/workflows/test.yml

```diff
@@ -43,10 +43,8 @@ jobs:
         python -m pip install --upgrade pip
         python -m pip install tox
     - name: tox
-      env:
-        TOXENV: ${{ matrix.toxenv }}
       run: |
-        tox -e py
+        tox -e ${{ matrix.toxenv || 'py' }}
```

Comment:
These changes were necessary since before, GitHub actions was not using some of the pinned configurations of `min`, `asyncio`, and `asyncio-min`.

For example, `min` was not using `scrapy==2.6.0` but the latest version at the current time: https://github.com/scrapinghub/scrapy-poet/actions/runs/4047094013/jobs/6960692434

Reply:
Moved this into a separate PR: #129

Reply:
The fix here was to change `tox -e py` to `tox`, then `TOXENV` should work as expected.

Reply:
I've gotten inspiration from [https://github.com/scrapy-plugins/scrapy-zyte-api/blob/main/.github/workflows/test.yml#L20-L48.](https://github.com/scrapy-plugins/scrapy-zyte-api/blob/main/.github/workflows/test.yml#L20-L48.)

Reply:
Hmm… Well, the -e approach now does allow to avoid quite some duplicate lines, so you are right, it is probably the best approach.